sdk/ml/azure-ai-ml/azure/ai/ml/constants/_common.py

```diff
@@ -562,3 +562,8 @@ class RollingRate:
     DAY = "day"
     HOUR = "hour"
     MINUTE = "minute"
+
+
+class Scope:
```

Comment:
Thx for creating constants for this, can you update worspace_operations to ues these constants too?

Reply:
Done, the only operation that needs it is list, much like registries.